Classification/SGD/src/main/java/org/tribuo/classification/sgd/objectives/Hinge.java

```diff
@@ -52,14 +52,20 @@ public Hinge() {
         this(1.0);
     }
 
+    @Deprecated
+    @Override
+    public Pair<Double, SGDVector> valueAndGradient(int truth, SGDVector prediction) {
+        return lossAndGradient(truth, prediction);
+    }
+
     /**
      * Returns a {@link Pair} of {@link Double} and {@link SparseVector}.
```

Comment:
The javadoc here seems a bit obvious? Feels like it should say the meaning of the Double and SparseVector you're returning? And for what purpose? Of course, that javadoc can be in the parent class, as we're overriding here.

Reply:
Yeah, the interface has better javadoc. I didn't actually change it on this method, but I can tidy it up.

Reply:
Fixed.